modules/storages/lib/open_project/storages/engine.rb

```diff
@@ -85,6 +85,24 @@ class Engine < ::Rails::Engine
            { controller: '/storages/admin/projects_storages', action: 'index' },
            caption: :project_module_storages,
            parent: :settings
+
+      configure_menu :project_menu do |menu, project|
```

Comment:
is this the first time a module adds a menu entry? Or why did you needed a new method for that?

Reply:
Yes, I needed a new method for that. It behaves differently from `menu` or `add_menu_item`. The difference is a possibility to depend on a project. for example.